glossary.asciidoc

```diff
@@ -170,11 +170,11 @@ Gossip Protocol::
     Lightning Network nodes send and receive information about the topology of the Lightning Network through gossip messages which are exchanged with their peers.
     The gossip protocol is mainly defined in BOLT 7 and defines the format of the _node_announcement_, _channel_announcement_ and _channel_update messages_.
     In order to prevent SPAM, node announcement messages will only be forwarded if the node already has a channel and channel announcement messages will only be forwarded if the funding transaction of the channel has been confirmed by the Bitcoin network.
-    Usually Lightning nodes connect with their channel partners, but it is fine connect with any other Lightning node in order to process gossip messages.
+    Usually Lightning nodes connect with their channel partners, but it is fine connected with any other Lightning node in order to process gossip messages.
```

Comment:
My initial typo. It was supposed to be "but it is fine to connect...." so the initial wird is correct but there is a word missing